pkg/controller/controller_test.go

```diff
@@ -1107,6 +1148,31 @@ func testNumberOfActions(t *testing.T, name string, f failfFunc, actions []clien
 	return true
 }
 
+func assertActionsVerb(t *testing.T, actions []clientgotesting.Action, verb string) {
+	testActionsVerb(t, "" /* name */, fatalf, actions, verb)
+}
+
+func expectActionsVerb(t *testing.T, name string, actions []clientgotesting.Action, verb string) bool {
```

Comment:
I still see the value of such generic method for making sure that only `get` requests were performed (no side-effects), for example.